admin-base/core/src/main/java/com/peregrine/admin/resource/AdminResourceHandlerService.java

```diff
@@ -436,9 +431,10 @@ private void copyAppsComponentToNewTemplate(String component, Node template) thr
     @Override
     public DeletionResponse deleteResource(ResourceResolver resourceResolver, String path, String primaryType) throws ManagementException {
         final Resource resource = getResource(resourceResolver, path);
-        if (resource == null) {
+        if (isNull(resource)) {
```

Comment:
Whats the gain from this?

Reply:
Well, cognitive ease - one can read it like a sentence. I have nothing against algebraic formulas, having math degree, but it's simply easier to read and e.g. not miss the subtle visual difference between `==` and `!=`. It might sound silly, but you know - the less one has to think about irrelevant stuff, the more brain power can be used for the core problem.

Though I'm going to minimize these kind of changes in any code that I write now, people don't want refactoring, got to respect that. So only when it's really needed from now on (not counting existing branches with no PRs yet).